janus_server/src/bin/aggregator.rs

```diff
@@ -35,6 +40,9 @@ async fn main() -> Result<()> {
     // TODO(issue #20): We need to specify configuration parameters rather than hardcoding them.
     let task_id = TaskId::random();
 
+    let vdaf = Prio3Aes128Count::new(2).unwrap();
+    let verify_param = vdaf.setup().unwrap().1.first().unwrap().clone();
```

Comment:
It's going to take some time before this is ready to merge (the dependent `libprio-rs` PR has at least one round of communication with cjpatton to go before it can possibly be merged), so let's rebase based on whoever gets to `main` first :-)